src/API.CRUD/Startup.cs

```diff
@@ -1,50 +1,56 @@
-﻿using API.CRUD.ActionFilters;
+﻿using System;
+using API.CRUD.Extensions;
 using API.CRUD.Repositories;
-using API.CRUD.Validators;
-using FluentValidation.AspNetCore;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
-using Swashbuckle.AspNetCore.Swagger;
+using Microsoft.Extensions.Logging;
+using StackifyMiddleware;
 
 namespace API.CRUD
 {
     public class Startup
     {
-        public Startup(IConfiguration configuration)
+        private const string CorsPolicyName = "Default";
+        private readonly ILogger _logger;
+            
+        public Startup(IConfiguration configuration,
+                       ILoggerFactory loggerFactory)
         {
-            Configuration = configuration;
+            Configuration = configuration ?? throw new ArgumentNullException(nameof(configuration));
+
+            if (loggerFactory == null)
+            {
+                throw new ArgumentNullException(nameof(loggerFactory));
+            }
+            _logger = loggerFactory.CreateLogger<Startup>();
+
+            _logger.LogInformation($" <::::[]==0 ------------------------------------------ o==[]::::> {Environment.NewLine} *** Starting WebApi.");
         }
 
-        public IConfiguration Configuration { get; }
+        private IConfiguration Configuration { get; }
 
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddMvcCore(config => // basic API stuff
-                    {
-                        config.Filters.Add(new ValidateModelAttribute());
-                    })
-                    .AddApiExplorer() // for swagger
-                    .AddFluentValidation(options => // for validation
-                    {
-                        options.RegisterValidatorsFromAssemblyContaining<Startup>();
-                        options.RunDefaultMvcValidationAfterFluentValidationExecutes = false;
-                    })
-                    .AddJsonFormatters(); // for JSON responses

```

Comment:
For discussion => why remove this? I _thought_ this does CamelCase/ISO-8601 stuff, which you seem to have done manually now above?

Reply:
We need lines 32/33. Fine to remove line 35.

Reply:
Was moved to an extension method : `AddCustomFluentValidation`.